core/trino-main/src/main/java/io/trino/sql/planner/LocalExecutionPlanner.java

```diff
@@ -2787,7 +2787,7 @@ private JoinBridgeManager<?> createLookupSourceFactoryManager(
 
```

Comment:
Do we need any change to `createLookupJoin` as well ?

Reply:
The `useSpillingJoinOperator` is used in this method already. The problem was that the creation of HashBuilder operator was not consistent with the creation of LookupJoin operator regarding spilling.
Now they use the same `useSpillingJoinOperator` method